code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -120,22 +128,25 @@
  */
 /datum/heretic_knowledge/rust_regen/proc/on_life(mob/living/source, seconds_per_tick, times_fired)
 	SIGNAL_HANDLER
-
 	var/turf/our_turf = get_turf(source)
 	if(!HAS_TRAIT(our_turf, TRAIT_RUSTY))
 		return
-
+	//damages non heretics' clothes and gives them a negative moodlet
```

Comment:
Seems to not be implemented yet but I am not convinced this is a good idea, equipment damage is a wildly powerful effect and heretic rust can cover quite significant areas of the station. If the heretic spaces those areas they have essentially made them impossible to cross as they will decay your space protection.

Reply:
what if it just destroys shoes

Reply:
bit weird but probably more ok?

Reply:
i want it to atleast destroy shoes because then they can fall on glass shards and b ottles in maintenence